.github/workflows/ttc.yml

```diff
@@ -0,0 +1,35 @@
+# .github/workflows/docker-publish.yml
+
+name: Docker
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'core/**'
+      - 'driver/**'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
@everpcpc  help me review these and add these configs into GitHub repo's settings

Reply:
cc @BohuTANG could you help with this?